### PR TITLE
Zivid sdk 1.4.0 version bump

### PR DESCRIPTION
**Pull request by nedrebo**

### CODE REVIEW

**Comment by nedrebo** [comment]
This pr also includes some older commits, see description per commit.

The main changes are:

1. Bump to 1.4.0
2. It is better to depend on the meta package opencl-driver than to opt-depend on a random and incomplete list of packages that provide OpenCL drivers.

File: continuous-integration/setup.sh [hunk 1]
```diff
@@ -12,6 +12,7 @@ pacman -Syu --noconfirm --needed \
        file \
        flake8 \
        grep \
+       intel-compute-runtime \
```
Comment (knatten) [inline continuous-integration/setup.sh]: Did you intend to commit this in the "Bump Zivid SDK version to 1.4.0" commit?

File: scrips/generate_pkgbuild.py [hunk 2]
```diff
@@ -25,22 +25,11 @@ class Pkgbuild:
 
     dependencies = {
         "zivid-telicam-sdk": (),
-        "zivid": ("zivid-telicam-sdk",),
+        "zivid": ("zivid-telicam-sdk", "opencl-driver"),
```
Comment (knatten) [inline scrips/generate_pkgbuild.py]: What's this `opencl-driver`?

Comment (nedrebo) [reply scrips/generate_pkgbuild.py]: Meta package, opencl-nvidia for example fulfills this dependency.

File: continuous-integration/setup.sh (hunk 1 above)
Comment (nedrebo) [reply continuous-integration/setup.sh]: No it should be part of 9416b00cd0d035e59e89902f6148c0c0acb02035, will fix.

Comment (nedrebo) [reply continuous-integration/setup.sh]: Fixed

**Pull request merged**